CuratedContent/ApplyingFAIRtoWorkflows.md

```diff
@@ -0,0 +1,40 @@
+# Applying the FAIR Principles to Computational Workflows
+<!--deck text start-->
+
+Whether your computational workflow employs a small set of bash scripts,
+or a digital notebook on your laptop, or is a complex workflow that
+utilizes a workflow management system (WMS) on High Performance Computing (HPC) resources,
+managing the resulting digital objects is crucial for promoting greater transparency and reproducibility.
+This article explains the components of computational workflows and the benefits of applying Findable, Accessible, Interoperable and Reusable (FAIR) data and software principles to the workflows
+.
+
+
+
+
+<!--deck text end-->
+
+#### Contributed by [Patricia Grubel](https://github.com/pagrubel "Patricia Grubel")
+#### Publication date: April 24, 2025
+
+Resource information | Details
+:--- | :---
+Article Title | [Applying the FAIR Principles to Computational Workflows](https://www.nature.com/articles/s41597-025-04451-9)
+Authors | Sean R. Wilkinson et al.
+Focus | FAIR Computational Workflows
+
+[Applying the FAIR Principles to Computational Workflows](https://www.nature.com/articles/s41597-025-04451-9) addresses both FAIR data and software principals for computational workflows, with specific recommendations applicable to workflows for workflow authors, WMS developers, registries and repositories.
+The Workflows Community Initiative’s FAIR Workflows Working Group (WCI-FW), a global and open community of researchers and developers working with computational workflows across disciplines and domains, explains the application of FAIR principles to computational workflows.
+
+As computation workflows are becoming increasingly complex and automated, while the underlying systems are abstracted by higher level languages, tools and management systems, it becomes essential to apply FAIR principles to computational workflows.
```

Comment:
Suggest changing the beginning of this sentence to:

"As computational workflows grow increasingly complex..."